doc/feature/pointDataEvolution.md

```diff
@@ -0,0 +1,123 @@
+Evolution of data at a point over time
+======================================
+
+One of the features that would make the tool extremely useful for scientists, is
```

Comment:
In fact, you're creating a new tool, so there's no single tool. So I'd say: One feature that would be extremely useful for scientists is the ...